gentoo/gentoo
# gnome-base/librsvg: Version bump to 2.56.4

Test result, same as 2.56.3
```
test errors::instancing_limit1 ... ok

failures:

---- reference::tests::bugs_bug668_small_caps_svg stdout ----
bug668-small-caps: 3551 pixels changed with maximum difference of 255
out: /var/tmp/portage/gnome-base/librsvg-2.56.3/work/librsvg-2.56.3-abi_x86_64.amd64/tests/output/bug668-small-caps-out.png
diff: /var/tmp/portage/gnome-base/librsvg-2.56.3/work/librsvg-2.56.3-abi_x86_64.amd64/tests/output/bug668-small-caps-diff.png
thread 'reference::tests::bugs_bug668_small_caps_svg' panicked at 'surfaces are too different', tests/src/reference_utils.rs:86:25
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace


failures:
    reference::tests::bugs_bug668_small_caps_svg

    test result: FAILED. 977 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 2.75s

    error: test failed, to rerun pass `--test src`
    make[3]: *** [Makefile:1541: check-local] Error 101
    make[3]: Leaving directory '/var/tmp/portage/gnome-base/librsvg-2.56.3/work/librsvg-2.56.3-abi_x86_64.amd64'
    make[2]: *** [Makefile:1379: check-am] Error 2
    make[1]: *** [Makefile:1083: check-recursive] Error 1
    make: *** [Makefile:1381: check] Error 2
     * ERROR: gnome-base/librsvg-2.56.3::gentoo failed (test phase):
     *   Make check failed. See above for details.
     * 
       * Call stack:
       *            ebuild.sh, line  136:  Called src_test
       *          environment, line 3830:  Called multilib-minimal_src_test
       *          environment, line 2660:  Called multilib_foreach_abi 'multilib-minimal_abi_src_test'
       *          environment, line 2868:  Called multibuild_foreach_variant '_multilib_multibuild_wrapper' 'multilib-minimal_abi_src_test'
       *          environment, line 2550:  Called _multibuild_run '_multilib_multibuild_wrapper' 'multilib-minimal_abi_src_test'
       *          environment, line 2548:  Called _multilib_multibuild_wrapper 'multilib-minimal_abi_src_test'
       *          environment, line  558:  Called multilib-minimal_abi_src_test
       *          environment, line 2656:  Called default_src_test
       *   phase-functions.sh, line  901:  Called __eapi0_src_test
       *     phase-helpers.sh, line  832:  Called die
       * The specific snippet of code:
       *              ${emake_cmd} ${internal_opts} check || \
       *                      die "Make check failed. See above for details."
       * 
       * If you need support, post the output of `emerge --info '=gnome-base/librsvg-2.56.3::gentoo'`,
       * the complete build log and the output of `emerge -pqv '=gnome-base/librsvg-2.56.3::gentoo'`.
       * The complete build log is located at '/var/tmp/portage/gnome-base/librsvg-2.56.3/temp/build.log'.
       * The ebuild environment file is located at '/var/tmp/portage/gnome-base/librsvg-2.56.3/temp/environment'.
       * Working directory: '/var/tmp/portage/gnome-base/librsvg-2.56.3/work/librsvg-2.56.3-abi_x86_64.amd64'
       * S: '/var/tmp/portage/gnome-base/librsvg-2.56.3/work/librsvg-2.56.3'

```

```

test errors::instancing_limit1 ... ok

failures:

---- reference::tests::bugs_bug668_small_caps_svg stdout ----
bug668-small-caps: 3551 pixels changed with maximum difference of 255
out: /var/tmp/portage/gnome-base/librsvg-2.56.4/work/librsvg-2.56.4-abi_x86_64.amd64/tests/output/bug668-small-caps-out.png
diff: /var/tmp/portage/gnome-base/librsvg-2.56.4/work/librsvg-2.56.4-abi_x86_64.amd64/tests/output/bug668-small-caps-diff.png
thread 'reference::tests::bugs_bug668_small_caps_svg' panicked at 'surfaces are too different', tests/src/reference_utils.rs:86:25
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace


failures:
    reference::tests::bugs_bug668_small_caps_svg

    test result: FAILED. 977 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 2.80s

    error: test failed, to rerun pass `--test src`
    make[3]: *** [Makefile:1541: check-local] Error 101
    make[3]: Leaving directory '/var/tmp/portage/gnome-base/librsvg-2.56.4/work/librsvg-2.56.4-abi_x86_64.amd64'
    make[2]: *** [Makefile:1379: check-am] Error 2
    make[1]: *** [Makefile:1083: check-recursive] Error 1
    make: *** [Makefile:1381: check] Error 2
     * ERROR: gnome-base/librsvg-2.56.4::gentoo failed (test phase):
     *   Make check failed. See above for details.
     * 
     * Call stack:
     *            ebuild.sh, line  136:  Called src_test
     *          environment, line 3830:  Called multilib-minimal_src_test
     *          environment, line 2660:  Called multilib_foreach_abi 'multilib-minimal_abi_src_test'
     *          environment, line 2868:  Called multibuild_foreach_variant '_multilib_multibuild_wrapper' 'multilib-minimal_abi_src_test'
     *          environment, line 2550:  Called _multibuild_run '_multilib_multibuild_wrapper' 'multilib-minimal_abi_src_test'
     *          environment, line 2548:  Called _multilib_multibuild_wrapper 'multilib-minimal_abi_src_test'
     *          environment, line  558:  Called multilib-minimal_abi_src_test
     *          environment, line 2656:  Called default_src_test
     *   phase-functions.sh, line  901:  Called __eapi0_src_test
     *     phase-helpers.sh, line  832:  Called die
     * The specific snippet of code:
     *              ${emake_cmd} ${internal_opts} check || \
     *                      die "Make check failed. See above for details."
     * 
     * If you need support, post the output of `emerge --info '=gnome-base/librsvg-2.56.4::gentoo'`,
     * the complete build log and the output of `emerge -pqv '=gnome-base/librsvg-2.56.4::gentoo'`.
     * The complete build log is located at '/var/tmp/portage/gnome-base/librsvg-2.56.4/temp/build.log'.
     * The ebuild environment file is located at '/var/tmp/portage/gnome-base/librsvg-2.56.4/temp/environment'.
     * Working directory: '/var/tmp/portage/gnome-base/librsvg-2.56.4/work/librsvg-2.56.4-abi_x86_64.amd64'
     * S: '/var/tmp/portage/gnome-base/librsvg-2.56.4/work/librsvg-2.56.4'
```
